packages/editor/src/components/block-list/block.js

```diff
@@ -409,16 +408,18 @@ export class BlockListBlock extends Component {
 						isParentOfSelectedBlock,
 						isDraggable,
 						className,
+						blockName,
+						isValid,
```

Comment:
It seems odd our mixed application of "block" as a prefix in props, when in fact the `isValid` is specifically referring to the validity of a block. Given that the component is "BlockListBlock", an argument could be made that the prefix is not necessary at all.

Not really a **block**er.

Reply:
> Not really a **block**er.

I love it :) I think I'll update and remove the prefix (I hope I won't have eslint issues :) )